templates/index.html

```diff
@@ -141,7 +141,7 @@ <h2 class="feature-title">Sound</h2>
                     Load audio files and play them on demand
                     <ul class="feature-sublist">
                         <li>Load audio files as Assets</li>
-                        <li>Play audio Assets using the Audio resource</li>
+                        <li>Play audio Assets using Audio bundles</li>
```

Comment:
```suggestion
                        <li>Play audio Assets using Audio entities</li>
```
I think this is probably more useful as a framing.